main.tf

```diff
@@ -156,7 +156,8 @@ resource aws_ecs_service service {
 
     content {
       registry_arn   = service_registries.value.arn
-      port           = var.service_discovery_container_port
+      # Setting port is not supported whne using "host" or "bridge" network mode.
```

Comment:
Mind removing the inline comment here, I think the ternary is pretty self explanatory